config/admin.config.php

```diff
@@ -131,6 +156,12 @@
             'collection_query_whitelist' => array('version'),
         ),
     ),
+    'zf-rpc' => array(
+        'ZF\Apigility\Doctrine\Admin\Controller\DbAutodiscovery' => array(
```

Comment:
I don't understand this like.  Can you explain?  It looks fine but because the DbAutodiscovery isn't a controller I want to clarify with@jguittard